flang/lib/Lower/ConvertExpr.cpp

```diff
@@ -568,20 +568,24 @@ class ExprLowering {
   mlir::Value genArrayLit(
       const Fortran::evaluate::Constant<Fortran::evaluate::Type<TC, KIND>>
           &con) {
-    // TODO:
-    // - Multi-dimensional array
-    if (con.Rank() > 1) {
-      TODO(); // "Multi dimensional array not yet supported"
-    }
-    fir::SequenceType::Shape shape(1, con.shape()[0]);
+    fir::SequenceType::Shape shape;
+    // Can't use std::vector constructor(Fortran::evaluate::ConstantSubscripts)
+    // to construct llvm::SmallVector (fir::SequenceType::Shape)
```

Comment:
No, you can't. It doesn't really make sense. A `SequenceType` is a FIR concept. An `Ev::ConstantSubscripts` is a front-end data structure. The latter has to be lowered into FIR.

Reply:
I agree. Do you think I should remove this comment?

Reply:
Yes. The lowering _is_ trivial, of course. I guess what I should've said is that it might be nice to have a function here that documents what is happening, even though it is pretty boring. Or, if not a function, then a comment like:
```
// convert Ev::ConstantSubs to SequenceType::Shape
```
I think you can do it in a one-liner with
```
fir::SequenceType::Shape shape(con.begin(), con.end());
```